src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -344,7 +344,7 @@ class ActionsPanel extends React.PureComponent<Props> {
               icon={<IconAdd isCircled color="gray500" />}
               onClick={this.handleAddAction}
             >
-              Add Item
+              Add New Action
```

Comment:
oh shoot this should be in a translate block

Reply:
Oh dang. I can fix that.